conformity/fields/email.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import, unicode_literals
+
+import re
+
+import attr
+
+from conformity.error import Error
+from conformity.fields.basic import UnicodeString
+from conformity.utils import strip_none
+from conformity.fields.net import (
```

Comment:
No need to indent this import I think, there's only one item.

Reply:
edited.